src/rust/src/x509/verify.rs

```diff
@@ -282,7 +282,7 @@ impl PyClientVerifier {
             policy,
             store.raw.borrow_dependent(),
         )
-        .map_err(|e| VerificationError::new_err(format!("validation failed: {e:?}")))?;
+        .map_err(|e| VerificationError::new_err(format!("validation failed: {e}")))?;
```

Comment:
NB: This will make all of the Python-surfaced errors much nicer, now that we have `Display for VerificationError`.